syntax = "proto3";

service Moeda {
  rpc converter (ConverterRequest) returns (ConverterReply) {}
}

message ConverterRequest {
  string moeda_origem = 1;
  double valor_origem = 2;
  string moeda_destino = 3;
}

message ConverterReply {
  double valor_origem = 1;
  double valor_destino = 2;
}